tests/fixtures/rules/require-const-for-ember-properties/good/member.js

```diff
@@ -0,0 +1,3 @@
+// jscs:disable requireObjectDestructuring
+const isNone = Ember.isNone;
```

Comment:
Since I merged the `disallowDirectPropertyAccess` rule, this test will now fail as `Ember.isNone` is not allowed. You'll need to disable the rule here as well:

```js
// jscs:disable disallowDirectPropertyAccess
```